packages/Molecules/Form/index.vue

```diff
@@ -66,10 +66,24 @@ export default {
   },
   computed: {
     submitLoaderFields () {
-      return this.formData.schema.fields.filter(item => (item.type === 'rplsubmitloader' && item.autoUpdate))
+      const fields = []
+      if (this.formData) {
+        if (this.formData.schema.groups) {
+          this.formData.schema.groups.forEach(group => {
+            fields.push(...group.fields.filter(this.isAutoUpdatedSubmitField))
```

Comment:
I still feel this loader button can be reviewed some point future for better design, as it's coupled tightly with from component, these vfg `groups` has been refactored in next release, that makes harder to upgrade vfg in the future.